crud/common/utils.lua

```diff
@@ -181,6 +181,9 @@ local function determine_enabled_features()
 
     -- since Tarantool 2.4
     enabled_tarantool_features.uuids = major >= 2 and (minor > 4 or minor == 4 and patch >= 1)
+
+    -- since Tarantool 2.0
+    enabled_tarantool_features.jsonpaths = major >= 2
```

Comment:
Since 2.6.3/2.7.2/2.8.1 because of https://github.com/tarantool/tarantool/pull/5497

Reply:
Here we should divide "jsonpath filters" that are available since 1.10 and "json path indexes" that was introduced in 2.1 but you can't it use in correct way because of a bug specified above.